src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -335,7 +335,9 @@ function ReimbursementAccountPage({reimbursementAccount, route, onfidoToken, pol
     );
 
     const continueFunction = () => {
-        setShouldShowContinueSetupButton(false);
+        BankAccounts.setBankAccountSubStep(CONST.BANK_ACCOUNT.SETUP_TYPE.MANUAL).then(() => {
```

Comment:
I made it as a promise so the sub-step is updated before `shouldShowContinueSetupButton` state to prevent the blink between the start and 1st step page as shown in the [video](https://github.com/Expensify/App/issues/26631#issuecomment-1881317276).